components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.call.ui/src/main/resources/web/call-mediator/update-mediator.jsp

```diff
@@ -79,6 +79,9 @@
         callMediator.setEndpoint(ep);
     }
 
+    boolean blocking=Boolean.parseBoolean(request.getParameter("mediator.call.blocking"));
```

Comment:
The "mediator.call.blocking" constant can be moved as a constant variable. I left because to be consistent with other mediators. I followed the standard in the PRs[1]  which I sent recently. I will check and fix
[1] - https://github.com/wso2/esb-connectors/pull/292